pkg/metrics/version.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2025 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/karmada-io/karmada/pkg/version"
+)
+
+// NewCollector returns a collector that exports metrics about current version
+// information.
+func NewCollector(program string) prometheus.Collector {
```

Comment:
```suggestion
func NewBuildInfoCollector(program string) prometheus.Collector {
```
At least we can use a more straightforward name which I suggested at https://github.com/karmada-io/karmada/pull/6044#discussion_r1976809533.

Reply:
Fixed

Reply:
Seems not :)

Reply:
> Seems not :)

Done